components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -152,6 +153,7 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
                     color={'inherit'}
                     role={'button'}
                     className={clsx(combine('rightComponent'), defaultClasses.flipIcon)}
+                    style={{ color: Colors.gray[500] }}
```

Comment:
What's the color supposed to be in the dark theme? Will this fix work for light & dark?

Reply:
were using `black[200]` for dark theme... aka `theme.palette.text.secondary`